src/stories/6-Modal.stories.js

```diff
@@ -1,36 +1,80 @@
 import React from "react";
 import Modal from "../components/Modal";
-// import { action } from "@storybook/addon-actions";
+import DurationBar from "../components/DurationBar";
+import RangeBar from "../components/RangeBar";
+import { action } from "@storybook/addon-actions";
+import styled from "styled-components";
+import Badge from "../components/Badge";
 
 export default {
   title: "Modal-Components"
 };
 
+const BadgeContainer = styled.div`
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+`;
+
+const Title = styled.h4`
+  color: grey;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Try to use a global style. It will result in less maintenance and you will have more control over your components.